packages/lodestar/src/interop/keypairs.ts

```diff
@@ -19,9 +19,8 @@ export function interopKeypairs(validatorCount: number): IKeypair[] {
 }
 
 export function interopKeypair(index: number): IKeypair {
-  const privkey = bytesToBN(hash(intToBytes(index, 32)))
-    .mod(CURVE_ORDER)
-    .toArrayLike(Buffer, "be", 32);
+  const privkey =
+    toBufferBE(bytesToBN(hash(intToBytes(index, 32))) % CURVE_ORDER, 32);
```

Comment:
intToBytes?

Reply:
this is the one big endian conversion

Reply:
oh yeah, thats wierd

Reply:
In the future I think it'd be ideal to replace BigInt type operations with constant time operations when we're handling sensitive keys like this.  The `toBufferLE` call that exists in `bytesToBN` [performs a couple String operations in addition to BigInt operations](https://github.com/no2chem/bigint-buffer/blob/76ed004c3a63bb031525f7b9d33ccb1bb24db919/src/index.ts#L25). 

This type of behavior could lead to timing attacks on keys depending on our execution environment (browser or OS), since Strings and [BigInts](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt) don't operate in constant time and aren't best for cryptography.

Reply:
It's an edge case at this point in time, but I think it should be on our radar for the future. If preferred, I can just open a low priority ticket for now.

Reply:
these are not secure keys, these are generate from 1...100 numbers for interop tests. We are in process of implementing bls specific way of generating secure bls private keys.

Reply:
Gotcha. Thanks for the clarification.